learningresources/models.py

```diff
@@ -27,6 +28,12 @@
 STATIC_ASSET_PREFIX = 'assets'
 STATIC_ASSET_BASEPATH = STATIC_ASSET_PREFIX + '/{org}/{course_number}/{run}/'
 
+# Used to signify that a repository has been created
+# and its initial permissions have been set.
+# This allows the "curation status" vocabulary to be
+# created automatically.
+repo_created = Signal(providing_args=["repository"])
```

Comment:
I created it here because it is called from this file (on the model `save()`). If there's a "correct" way to do it I don't know it. It just made sense to put it here. If I put it in signals then I'd have to import it here (as much code as creating it), so it feels okay to put it here.